### ISSUE TITLE
what is the masking ratio

### ISSUE THREAD

**Issue by abhidipbhattacharyya** [jerryji1993/DNABERT#119]
Hi,
First of all thank you for releasing such a clean code base. It is very helpful. I am trying to train a model from scratch. In the paper the masking probability is mentioned as 15%. But the example command in the readme file has it as 0.025(2.5%). Which one should I use? Are those hyper param mentioned in the training command in the readme file actual ones used during pretraining?
